crates/precompile/src/utilities.rs

```diff
@@ -26,8 +25,7 @@ pub fn right_pad<const LEN: usize>(data: &[u8]) -> Cow<'_, [u8; LEN]> {
         Cow::Borrowed(data.try_into().unwrap())
     } else {
         let mut padded = [0; LEN];
-        let end = min(LEN, data.len());
-        padded[..end].copy_from_slice(&data[..end]);
+        padded[..data.len()].copy_from_slice(data);
```

Comment:
I realized these `min` calls were unnecessary since `data.len()` is always less than the input length by definition.